pandas/core/_numba/kernels/min_max_.py

```diff
@@ -112,11 +112,9 @@ def grouped_min_max(
             continue
 
         if is_max:
-            if val > output[lab]:
-                output[lab] = val
+            output[lab] = max(val, output[lab])
         else:
-            if val < output[lab]:
-                output[lab] = val
+            output[lab] = min(val, output[lab])
```

Comment:
Are these maybe rules we want to ignore?

```python
def foo(a, b):
    b = max(a, b)

def bar(a, b):
    if a > b:
        b = a

%timeit foo(1, 2)
# 96.9 ns ± 0.84 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
%timeit bar(1, 2)
# 29.1 ns ± 0.151 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

%timeit foo(2, 1)
# 98.3 ns ± 0.42 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
%timeit bar(2, 1)
# 30.3 ns ± 0.173 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Reply:
Sure agreed, will revert this rule